### PR TITLE
feat: Add Avatars and Badges Components

### PR DESCRIPTION
**Pull request by rahulbakshi4**
Features Added:
1. Avatars
	- Basic Avatars
	- Avatars with different Sizes
	- Squared Avatars
2. Badges
	- Badges on Avatars
	- Badges on Icons

### CODE REVIEW

File: Components/Badges/badge.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    position: relative;
+}
+
+.state-div {
+    position: absolute;
+    bottom: 4px;
+    right: 2px;
+    border-radius: 50%;
+    width: 14px;
+    height: 14px;
```
Comment (romabulani) [inline Components/Badges/badge.css:11]: You can use rems here instead of px

File: Components/Avatars/avatar.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                        <a class="links" href="#">Colors</a>
+                    </li>
+                    <li>
+                        <a class="links" href="#">Typography</a>
+                    </li>
+                </ul>
+            </div>
+
+            <!-- Components List Items -->
+            <div class="">
```
Comment (romabulani) [inline Components/Avatars/avatar.html]: If class is not there, you can remove class = ""

File: Components/Badges/badge.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                        <a class="links" href="#">Colors</a>
+                    </li>
+                    <li>
+                        <a class="links" href="#">Typography</a>
+                    </li>
+                </ul>
+            </div>
+            
+            <!-- Component list Items -->
+            <div class="">
```
Comment (romabulani) [inline Components/Badges/badge.html]: Here also, you can remove empty classes

**Review by romabulani — APPROVED** [review]
Good work Rahul!! after changes, you can merge

File: Components/Badges/badge.css (hunk 1 above)
Comment (rahulbakshi4) [reply Components/Badges/badge.css:11]: I was avoiding the going for fractional value of rem like (0.85 rem) so used px.

File: Components/Avatars/avatar.html (hunk 2 above)
Comment (rahulbakshi4) [reply Components/Avatars/avatar.html]: Sure will make the changes

File: Components/Badges/badge.css (hunk 1 above)
Comment (romabulani) [reply Components/Badges/badge.css:11]: Ok, understood

File: Components/Badges/badge.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        <section class="doc-content">
+            <h3 class="fw-bold text-2xl">
+                Badges
+            </h3>
+            
+            <!-- Badges on Avatar -->
+            <div class="component-container">
+                <div class="badge">
+                    <img class="avatar"
+                        src="https://res.cloudinary.com/rahulb4/image/upload/v1643855031/peep_rssuj0.png" alt="" />
```
Comment (rohit8686) [inline Components/Badges/badge.html]: Everything looks good but do not keep alt empty

**Review by rohit8686 — APPROVED** [review]

File: Components/Badges/badge.html (hunk 4 above)
Comment (rahulbakshi4) [reply Components/Badges/badge.html]: Added missing alt tags

**Pull request merged**